### ISSUE TITLE
Zasoby /sale/disputes - udostępniliśmy statusy ostatnich wiadomości w dyskusjach / We have introduced new statuses of most recent messages in the discussions in /sale/disputes resources

### ISSUE THREAD

**Issue by Lukasz-Zurek** [allegro/allegro-api#4846]
W odpowiedzi na Wasze sugestie postanowiliśmy usprawnić proces odczytywania nowych wiadomości w dyskusjach. Dlatego od dziś zwracamy dwa nowe pola:

- **“messagesStatus”** - status ostatniej wiadomości w dyskusji,
- **“lastMessageCreationDate”** - data utworzenia ostatniej wiadomości.

**Na jakie zasoby mają wpływ te zmiany?**

- [GET /sale/disputes](https://developer.allegro.pl/documentation/#operation/getListOfDisputesUsingGET),
- [GET /sale/disputes/{disputeId}](https://developer.allegro.pl/documentation/#operation/getDisputeUsingGET).

Ponadto w polu **“messagesStatus”** zwrócimy następujące wartości:

- **“NEW”** - nowa dyskusja (w której sprzedający jeszcze nie odpowiedział),
- **“BUYER_REPLIED”** - nowa wiadomość od kupującego,
- **“SELLER_REPLIED”** - nowa wiadomość od sprzedającego,
- **“ALLEGRO_ADVISOR_REPLIED”** - nowa wiadomość od Allegro.

Więcej informacji na temat dyskusji w API znajdziesz w [naszym poradniku](https://developer.allegro.pl/dispute/#dyskusje).

----

In response to your suggestions, we decided to improve the process of reading new messages in discussions. Therefore, starting today, we are returning two new fields:

- **"messagesStatus"** - status of most recent message in the discussion,
- **"lastMessageCreationDate"** - most recent message creation date.

Which resources are affected by these changes?

- [GET /sale/disputes](https://developer.allegro.pl/en/documentation/#operation/getListOfDisputesUsingGET),
- [GET /sale/disputes/{disputeId}](https://developer.allegro.pl/en/documentation/#operation/getDisputeUsingGET).

In addition, in the **"messagesStatus"** field we will return the following values:

- **"NEW"** - new dispute (in which the seller has not responded yet),
- **"BUYER_REPLIED"** - new message from buyer,
- **"SELLER_REPLIED"** - new message from seller,
- **"ALLEGRO_ADVISOR_REPLIED"** - new message from Allegro.

You will find more information about API discussions in [our guide](https://developer.allegro.pl/en/dispute/#disputes).

**Comment by tomasz-grzesik** [comment]
Super.

**Comment by tomasz-grzesik** [comment]
Czy wiadomości są również sortowane po dacie lastMessageCreationDate?

**Comment by Lukasz-Zurek** [comment]
@tomasz-grzesik Nie, w tym endpoincie sortowanie jest tylko po dacie stworzenia dyskusji.

**Comment by tomasz-grzesik** [comment]
Szkoda.
